serving/samples/autoscale-go/README.md

```diff
@@ -89,7 +50,9 @@ Build the application container and publish it to a container registry:
 1. Ramp up traffic to maintain 10 in-flight requests.
 
    ```
-   go run serving/samples/autoscale-go/test/test.go -sleep 100 -prime 10000 -bloat 5 -qps 9999 -concurrency 300
+   docker run --rm -i -t --entrypoint /load-generator -e IP_ADDRESS="${IP_ADDRESS}" \
```

Comment:
What's the advantage / simplification of using `docker run` over `go run`?  Now I cannot run this example on my Pixelbook running Crostini because I cannot run containers locally.

I rather like being able to build, run and test things entirely in user-space.  E.g. using `ko` go build instead of `docker`.

Reply:
It's likely more people will have docker than go in this audience.

I just wanted to shorten the tutorial. We can probably make the docker command an `alias` and maybe use that here.